src/streaming/models/MediaPlayerModel.js

```diff
@@ -84,20 +85,32 @@ function MediaPlayerModel() {
     }
 
     /**
-     * Returns the maximum playback rate to be used when applying the catchup mechanism
+     * Returns the minium and maximum playback rates to be used when applying the catchup mechanism
+     * If only one of the min/max values has been set then the other will default to 0 (no playback rate change).
      * @return {number}
      */
-    function getCatchupPlaybackRate() {
-        if (!isNaN(settings.get().streaming.liveCatchup.playbackRate) && settings.get().streaming.liveCatchup.playbackRate > 0) {
-            return settings.get().streaming.liveCatchup.playbackRate;
+    function getCatchupPlaybackRates() {
```

Comment:
I think personally I'd not mix and match values from either location. I'd assume if an app specifies `playbackRate.max` it would be aware that `playbackRate.min` can also be set and has purposefully chosen not to, and would not expect it to be set from the MPD values. I could add a note in `settings.js` perhaps for clarity?

Reply:
A short note in the settings would be great, thank you